app/containers/settings/categories/GeneralSettingsPage.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React, { Component, PropTypes } from 'react';
+import { observer, inject } from 'mobx-react';
+import Request from '../../../stores/lib/Request';
+import GeneralSettings from '../../../components/settings/categories/GeneralSettings';
+
+@inject('stores', 'actions') @observer
+export default class GeneralSettingsPage extends Component {
+
+  static propTypes = {
+    actions: PropTypes.shape({
+      profile: PropTypes.shape({
+        updateLocale: PropTypes.func.isRequired,
+      }),
+    }),
```

Comment:
Please add `isRequired` to everywhere, so that potential problems are caught when things are wired wrongly